exchanges/exchange.go

```diff
@@ -1943,3 +1945,30 @@ func (b *Base) GetTradingRequirements() protocol.TradingRequirements {
 	}
 	return b.Features.TradingRequirements
 }
+
+// FetchTickerCached returns the ticker for a currency pair and asset type
+// associated with the exchange.
+// NOTE: UpdateTicker (or if supported UpdateTickers) method must be
+// called first to update the ticker map.
```

Comment:
This invalidate the naming for me because `Fetch*Cached` means "Fetch something with a cache around the fetch".
Assuming I've understood correctly.

Normally for me:
* Get* doesn't imply it must be an API call; It could be local or remote
* Fetch implies an API call or remote operation
* Cached means you're cache guarding it

So for this, I think it should be `GetCachedTicker`

Reply:
Addressed all of this I think, more nits please 🙏.